modules/module-mssql/src/common/mssqls-to-sqlite.ts

```diff
@@ -0,0 +1,151 @@
+import sql from 'mssql';
+import { DatabaseInputRow, ExpressionType, SqliteInputRow, toSyncRulesRow } from '@powersync/service-sync-rules';
+import { MSSQLUserDefinedType } from '../types/mssql-data-types.js';
+
+export function toSqliteInputRow(row: any, columns: sql.IColumnMetadata): SqliteInputRow {
+  let result: DatabaseInputRow = {};
+  for (const key in row) {
+    // We are very much expecting the column to be there
+    const columnMetadata = columns[key];
+
+    if (row[key] !== null) {
+      switch (columnMetadata.type) {
+        case sql.TYPES.BigInt:
+          // MSSQL returns BIGINT as a string to avoid precision loss
+          if (typeof row[key] === 'string') {
+            result[key] = BigInt(row[key]);
+          }
+          break;
+        case sql.TYPES.Bit:
+          // MSSQL returns BIT as boolean
+          result[key] = row[key] ? 1 : 0;
```

Comment:
Use 1n for true, and 0n for false - that is important to preserve these as integers rather than floating-point values throughout sync rules. Can use `SQLITE_TRUE` / `SQLITE_FALSE` constants for this.